float32[] A1
float32[] A2
float32[] b

int64 n

# Matrices so that Ax <= b is the area inside the polytope
# Matrix A is splitted in 2 column vectors A1 and A2
# note! in C++ the vectors go from 0 to n-1
